specification/compute/resource-manager/Microsoft.Compute/stable/2021-07-01/compute.json

```diff
@@ -8318,6 +8952,253 @@
+        }
+      },
+      "required": [
+        "value"
+      ],
+      "description": "The List capacity reservation group with resource group response."
+    },
+    "CapacityReservationInstanceView": {
+      "type": "object",
+      "properties": {
+        "utilizationInfo": {
+          "$ref": "#/definitions/CapacityReservationUtilization",
+          "description": "Unutilized capacity of the capacity reservation."
+        },
+        "statuses": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/InstanceViewStatus"
+          },
+          "description": "The resource status information."
+        }
+      },
+      "description": "The instance view of a capacity reservationthat provides as snapshot of the runtime properties of the capacity reservation that is managed by the platform and can change outside of control plane operations."
+    },
+    "CapacityReservationUtilization": {
+      "type": "object",
+      "properties": {
+        "virtualMachinesAllocated": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SubResourceReadOnly"
+          },
+          "readOnly": true,
+          "description": "A list of references to all virtual machines allocated in the Capacity reservation."
+        }
+      },
+      "description": "Represents the capacity reservation utilization in terms of resources allocated."
+    },
+    "CapacityReservationInstanceViewWithName": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The name of the capacity reservation."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/CapacityReservationInstanceView"
+        }
+      ],
+      "description": "The instance view of a capacity reservation that includes the name of the capacity reservation. It is used for the response to the instance view of a capacity reservation group."
+    },
+    "CapacityReservationProperties": {
+      "type": "object",
+      "properties": {
+        "reservationId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "A unique id generated and assigned to the capacity reservation by the platform. <br><br> Does not change throughout the lifetime of the capacity reservation."
+        },
+        "virtualMachinesAssociated": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SubResourceReadOnly"
+          },
+          "readOnly": true,
+          "description": "A list of references to all virtual machines associated with the capacity reservation."
+        },
+        "provisioningTime": {
+          "readOnly": true,
+          "type": "string",
+          "format": "date-time",
+          "description": "The date when the capacity reservation was last time updated."
```

Comment:
The date time when the capacity reservation was last updated.

Reply:
updated